scripts/v1alphaX/deprovision_cluster.sh

```diff
@@ -14,6 +14,10 @@ source "${METAL3_DIR}/lib/network.sh"
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/images.sh"
 
+if [[ "${1}" == "ug" ]]; then
```

Comment:
This would conflict with https://github.com/metal3-io/metal3-dev-env/pull/403 depending on which gets merged first

Reply:
same for all following files